package.json

```diff
@@ -88,7 +92,8 @@
       "web3-provider-engine>eth-sig-util>ethereumjs-abi>ethereumjs-util>keccak": false,
       "web3-provider-engine>eth-sig-util>ethereumjs-abi>ethereumjs-util>secp256k1": false,
       "web3-provider-engine>ethereumjs-vm>ethereumjs-util>keccak": false,
-      "web3-provider-engine>ethereumjs-vm>ethereumjs-util>secp256k1": false
+      "web3-provider-engine>ethereumjs-vm>ethereumjs-util>secp256k1": false,
+      "@swc/core": true
```

Comment:
I've enabled the script for `@swc/core` here, since it uses native packages to build the files.